samples/secret/karavi-authorization-config.json

```diff
@@ -0,0 +1 @@
+[{"username":"-","password":"-","intendedEndpoint":"https://10.0.0.1:8080","endpoint":"https://localhost:9400","systemID":"myIsilonClusterName","insecure":true,"isDefault":true}]
```

Comment:
I still see a key with a name insecure in this secret. It is renamed to skipCertificateValidation in helm/csi-isilon/values.yaml . Needs to be updated here as well?

Reply:
This change would require a code change in authorization since this field is defined in a [type](https://github.com/dell/karavi-authorization/blob/4549b8a9ef316905f910ff0db4c5892b63c06a27/cmd/sidecar-proxy/main.go#L74). We'd rather not have authorization code changes at this time but we have an item to address this in the future.